### PR TITLE
Added support for default file values, closes #256

### LINKED ISSUE #256: Add default file values syntax
as agreed at https://forum.opendatakit.org/t/proposal-specify-a-fixed-image-to-be-annotated-or-signed/14954/19

### PR DESCRIPTION
**Pull request by MartijnR**
Enketo should have an implementation in a few days if all goes well. I think it may make sense to do this for all 3 drawing widgets (and that would actually be slightly less work).

### CODE REVIEW

File: _sections/20-instance.md [hunk 1]
```diff
@@ -23,7 +23,7 @@ The _primary instance_ is the first instance defined by the form and should cont
 </instance>
 {% endhighlight %}
 
-Any value inside a primary instance is considered a default value for that question. If that node has a corresponding input element that value will be displayed to the user when the question is rendered.
+Any value inside a primary instance is considered a default value for that question. If that node has a corresponding input element that value will be displayed to the user when the question is rendered. For nodes of [type "binary"](http://localhost:4000/xforms-spec/#data-types), defaults use [file endpoint URIs](http://localhost:4000/xforms-spec/#file-endpoints).
```
Comment (lognaturel) [inline _sections/20-instance.md]: Strip the `localhost`, please! 😊

**Review by lognaturel — CHANGES_REQUESTED** [review]
Quick thing!

Collect will get support in v1.26 probably in Feb/March but let's merge now since the spec is agreed on. As written, ALL binary widgets should support this, right? Including audio/video/file? It doesn't seem particularly useful in those cases but seems that all should work?

File: _sections/20-instance.md (hunk 1 above)
Comment (MartijnR) [reply _sections/20-instance.md]: OMG.  Thanks!

**Comment by MartijnR** [comment]
Yes, I think all binary widgets from a spec perspective.

**Review by lognaturel — APPROVED** [review]

**Pull request merged**